Makefile

```diff
@@ -17,14 +17,20 @@ test-cov-report:
 integ-test:
 	pytest --no-cov integration/
 
-black:
+format:
 	black setup.py samtranslator tests integration bin schema_source
 	bin/transform-test-error-json-format.py --write tests/translator/output/error_*.json
 	bin/json-format.py --write tests integration samtranslator/policy_templates_data
 	bin/yaml-format.py --write tests
 	bin/yaml-format.py --write integration --add-test-metadata
 
-black-check:
+black:
+	$(warning `make black` is deprecated, please use `make format`)
+	# sleep for 5 seconds so the message can be seen.
+	sleep 5
```

Comment:
Please change CI as well or remove this.

Reply:
done

Reply:
good catch